projects/wave-core-new/src/lib/wave-core.module.ts

```diff
@@ -1,103 +1,104 @@
+import { SafeStylePipe } from './util/pipes/safe-style.pipe';
+import { ColormapNameToColorizerDataPipe } from './colors/colormap-colorizer/colormap-name-to-colorizer-data.pipe';
+import { SmallTimeInteractionComponent } from './time/small-time-interaction/small-time-interaction.component';
+import { TimeConfigComponent } from './time/time-config/time-config.component';
+import { TimeInputComponent } from './time/time-input/time-input.component';
+import { WorkspaceSettingsComponent } from './project/workspace-settings/workspace-settings.component';
+import { ChangeSpatialReferenceComponent } from './project/change-spatial-reference/change-spatial-reference.component';
+import { IfGuestDirective } from './util/directives/if-guest.directive';
+import { IfLoggedInDirective } from './util/directives/if-logged-in.directive';
+import { NewProjectComponent } from './project/new-project/new-project.component';
+import { LoadProjectComponent } from './project/load-project/load-project.component';
+import { SaveProjectAsComponent } from './project/save-project-as/save-project-as.component';
+import { MultiLayerSelectionComponent } from './operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component';
+import { OperatorListComponent } from './operators/dialogs/operator-list/operator-list.component';
+import { ExpressionOperatorComponent } from './operators/dialogs/expression-operator/expression-operator.component';
+import { OperatorOutputNameComponent } from './operators/dialogs/helpers/operator-output-name/operator-output-name.component';
+import { AddDataComponent } from './datasets/add-data/add-data.component';
+import { DatasetListComponent } from './datasets/dataset-list/dataset-list.component';
+import { DataSetComponent } from './datasets/dataset/dataset.component';
+import { PlotListComponent } from './plots/plot-list/plot-list.component';
+import { StatisticsPlotComponent } from './operators/dialogs/statistics-plot/statistics-plot.component';
+import { PlotDetailViewComponent } from './plots/plot-detail-view/plot-detail-view.component';
+import { PlotListEntryComponent } from './plots/plot-list-entry/plot-list-entry.component';
+import { HistogramOperatorComponent } from './operators/dialogs/histogram-operator/histogram-operator.component';
+import { LayerSelectionComponent } from './operators/dialogs/helpers/layer-selection/layer-selection.component';
+import { VegaViewerComponent } from './plots/vega-viewer/vega-viewer.component';
+import { LineageGraphComponent } from './provenance/lineage-graph/lineage-graph.component';
+import { MeanRasterPixelValuesOverTimeDialogComponent } from './operators/dialogs/mean-raster-pixel-values-over-time-dialog/mean-raster-pixel-values-over-time-dialog.component';
+import { PointInPolygonFilterOperatorComponent } from './operators/dialogs/point-in-polygon-filter/point-in-polygon-filter.component';
```

Comment:
no

Reply:
manually revert format by hand before we change the format again tomorrow: 009ab0a4c7897d1b68e9ee7e18fc7b6e600a9268